### PR TITLE
[FT:] Fix send alerts tests

### PR DESCRIPTION
**Pull request by miritih**
**Problem**
Send alerts tests were incomplete 

**Solution**
> **Fixed the following test:**
`test_it_handles_few` by asserting handle many commands returns true

> **Created the following new test**
`test_handle_many_true` and asserting that has many commands returns true

https://www.pivotaltracker.com/story/show/158729319

### CODE REVIEW

File: hc/api/tests/test_sendalerts.py [hunk 1]
```diff
@@ -37,6 +37,18 @@ def test_it_handles_grace_period(self):
         check.save()
 
         # Expect no exceptions--
-        Command().handle_one(check)
+        result=Command().handle_one(check)
+        self.assertEqual(True,result, "handle_one should return True")
 
-    ### Assert when Command's handle many that when handle_many should return True
```
Comment (thoth-ky) [inline hc/api/tests/test_sendalerts.py]: Remove prompts, otherwise good job

File: hc/api/tests/test_sendalerts.py [hunk 2]
```diff
@@ -37,6 +37,18 @@ def test_it_handles_grace_period(self):
         check.save()
 
         # Expect no exceptions--
-        Command().handle_one(check)
+        result=Command().handle_one(check)
```
Comment (thoth-ky) [inline hc/api/tests/test_sendalerts.py]: Space around `=`

File: hc/api/tests/test_sendalerts.py [hunk 3]
```diff
@@ -37,6 +37,18 @@ def test_it_handles_grace_period(self):
...
+        self.assertEqual(True,result, "handle_one should return True")
 
-    ### Assert when Command's handle many that when handle_many should return True
+    @patch("hc.api.management.commands.sendalerts.Command.handle_one")
+    def test_handle_many_true(self, mock):
+        """Assert when Command's handle many that when handle_many should return True"""
+        time = timezone.now() - timedelta(days=1)
+        names = ["Name %d" % n for n in range(0,1000)]
```
Comment (thoth-ky) [inline hc/api/tests/test_sendalerts.py]: space after `,`

File: hc/api/tests/test_sendalerts.py (hunk 2 above)
Comment (miritih) [reply hc/api/tests/test_sendalerts.py]: Fixed this

File: hc/api/tests/test_sendalerts.py (hunk 3 above)
Comment (miritih) [reply hc/api/tests/test_sendalerts.py]: fixed

**Review by thoth-ky — APPROVED** [review]
Good work

**Review by simiyu1 — APPROVED** [review]
Looks okay, prompts have been removed and the code is well commented.

**Review by loicemeyo — CHANGES_REQUESTED** [review]
Remove commented out line of code

**Review by loicemeyo — APPROVED** [review]

File: hc/api/tests/test_sendalerts.py [hunk 4]
```diff
@@ -37,6 +36,18 @@ def test_it_handles_grace_period(self):
...
+        """Assert when Command's handle many that when handle_many should return True"""
+        time = timezone.now() - timedelta(days=1)
+        names = ["Name %d" % n for n in range(0, 1000)]
+        for name in names:
+            check = Check(user=self.bob, name=name)
+            check.alert_after = time
+            check.status = "up"
+            check.save()
+        result = Command().handle_many()
+        self.assertEqual(result, True, "handle_many should return True")
```
Comment (karanjaE) [inline hc/api/tests/test_sendalerts.py]: missing empty line a the end of the file

**Review by karanjaE — CHANGES_REQUESTED** [review]

File: hc/api/tests/test_sendalerts.py (hunk 4 above)
Comment (miritih) [reply hc/api/tests/test_sendalerts.py]: I have rectified this @ranchow

**Comment by miritih** [comment]
I have resolved the conflicts

**Review by karanjaE — APPROVED** [review]

**Pull request merged**